src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/RelatedWorksArtworkGrid.tsx

```diff
@@ -23,16 +23,19 @@ const logger = createLogger("RelatedWorksArtworkGrid.tsx")
 
 const MAX_TAB_ITEMS = 3
 
-interface Props {
+interface RelatedWorksArtworkGridProps {
```

Comment:
any reason for this? sometimes the namespacing can be a bit...tedious. I've been using `Props` and `State` in the file, for the local container/component props and state.

Also semi-related to a PR discussion with @oxaudo on naming/organization of the various elements in a file (query renderer, fragment container, react component, props/state, occasional wrappers to add mediator and other things) - https://github.com/artsy/reaction/pull/1646#discussion_r239610662

Reply:
Mostly just for consistency with other code -- technically speaking the only time I naming needs to be this precise is when the code is being imported from another application where `d.ts` files are automatically generated.